packages/plans-grid-next/src/hooks/data-store/types.ts

```diff
@@ -26,6 +26,7 @@ export interface UseGridPlansParams {
 	 * Provide a map of plan slug keyed strings to display as the highlight label on top of each plan.
 	 */
 	highlightLabelOverrides?: { [ K in PlanSlug ]?: TranslateResult };
+	segmentationAnswers?: { [ key: string ]: string[] };
```

Comment:
Let's clarify what this property is and why it needs to be introduced in the grid. Also the signature is too open ended. Have you considered defining a more concrete type for the values?